apps/messagegui/app.js

```diff
@@ -505,7 +505,7 @@ function checkMessages(options) {
         // draw the body
         g.drawString(l.join("\n"), x+10,r.y+20+pady);
       }
-      if (!longBody && msg.src) g.setFontAlign(1,1).setFont("6x8").drawString(msg.src, r.x+r.w-2, r.y+r.h-2);
+      if (!longBody && msg.src) g.setFontAlign(-1,1).setFont("6x8").drawString(msg.src, r.x+r.w-2, r.y+r.h-2);
```

Comment:
Let's pull this out to another PR - if you could include @thyttan, @gfwilliams, @paul-arg and @rigrig, they're the main contributors so we can run it by them (it will also need a `metadata.json` and changelog bump)

(let me know if you need a hand)

Reply:
Yes - this one looks wrong to me anyway - if it's -1 that's left-aligned which means the start coordinate should be at the left of the box like `r.x+2`, not the right as in `r.x+r.w-2`

Reply:
Makes sense - I'll remove this in the merge and we can always add it back in later